CybercentreCanada/superset
# Feature/flow 8 get-eml functionality

### SUMMARY
In relation to this ticket: https://cccs.atlassian.net/browse/CLDN-2163
Users should be able to right-click on a row containing a path to an EML file on the Email Harmonized Dashboard and retrieve that file via a dropdown menu item within Ag-grid.  The download will start and inform the user if the download was successful or not.  

Deliverables:
A deployment of Superset that allows users to retrieve EML files.
A version of the Email Triaging Dashboard that allows users to retrieve EML files.

### TESTING INSTRUCTIONS
1.  Go on Superset-stg (Superset Image: feature_flow-8_20240319202431_b9607)
2.  In the main view, right click on a row and then click 'Download Email'
3. Download should start

### ADDITIONAL INFORMATION
- [ x] Required feature flags:
- [ x] Changes UI
